charts/datadog/templates/system-probe-configmap.yaml

```
@@ -13,14 +13,18 @@ metadata:
 data:
   system-probe.yaml: |
     system_probe_config:
-      enabled: {{ or $.Values.datadog.systemProbe.enabled $.Values.datadog.securityAgent.runtime.enabled }}
+      enabled: {{ or $.Values.datadog.systemProbe.enabled $.Values.datadog.securityAgent.runtime.enabled $.Values.datadog.networkMonitoring.enabled }}
```

why not using `{{- if eq (include "should-enable-system-probe" .) "true" }}` here too

---

Also since in line1 we already surround  the Configmap with `{{- if eq (include "should-enable-system-probe" .) "true" }}` 
we should be able to just set the value to `true`

```suggestion
      enabled: true
```